interop/xds/client/client.go

```diff
@@ -47,12 +47,13 @@ type statsWatcher struct {
 }
 
 var (
-	numChannels   = flag.Int("num_channels", 1, "Num of channels")
-	printResponse = flag.Bool("print_response", false, "Write RPC response to stdout")
-	qps           = flag.Int("qps", 1, "QPS per channel")
-	rpcTimeout    = flag.Duration("rpc_timeout", 10*time.Second, "Per RPC timeout")
-	server        = flag.String("server", "localhost:8080", "Address of server to connect to")
-	statsPort     = flag.Int("stats_port", 8081, "Port to expose peer distribution stats service")
+	failOnFailedRpc = flag.Bool("fail_on_failed_rpc", false, "Fail client if any RPCs fail")
```

Comment:
lint: var failOnFailedRpc should be failOnFailedRPC

Reply:
Fixed